### ISSUE TITLE
Add F3 partipation % to the metircs board

### ISSUE THREAD

**Issue by jennijuju** [filecoin-project/go-f3#789]
- add a graph that shows what is the % of the miner IDs are participating in f3
- add a table to list miner IDs that are not participating in f3

**Comment by masih** [comment]
**Notes on how to approach this**

First one becomes possible to do in Grafana once either of following is done:
* https://github.com/filecoin-project/go-f3/issues/745
* https://github.com/filecoin-project/go-f3/issues/728

The second one needs at least #728. Then a query to add to the existing Observer dashboad to get the diff of observed senders from power table at instance (after we schema is expanded to indeed store power table entries and power per instance).

**Comment by BigLep** [comment]
Per 2025-03-19 standup, we expect to handle as part of https://github.com/filecoin-project/go-f3/issues/728

**Comment by BigLep** [comment]
@masih : this is unblocked now right?

**Comment by masih** [comment]
> add a graph that shows what is the % of the miner IDs are participating in f3

This is done.

> add a table to list miner IDs that are not participating in f3

This one ideally requires multiple sources: Observer and signers of finality certs. Because looking at each alone might give us high rate of false negatives. To get this item done https://github.com/filecoin-project/go-f3/issues/745 needs to be done first. I will break it out of its own issue.

**Comment by masih** [comment]
Separated non-participating miner ID measurements here: https://github.com/filecoin-project/go-f3/issues/952

Closing this issue.
